internal/topology/variables/util.go

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
let's rename the file into json_shema.go or simply schema.go (let's avoid util if possible)